custom_components/tuya_local/devices/inkbird_ibs_m1s.yaml

```diff
@@ -0,0 +1,326 @@
+name: Inkbird IBS-M1S hub
+products:
+  - id: ojlc1dnjbrybk7di
+    name: IBS-M1S
+primary_entity:
+  entity: sensor
+  name: Temperature probe 1
+  class: temperature
+  dps:
+    - id: 110
+      type: integer
+      name: dummy
+      persist: false
```

Comment:
`persist: false` should only be needed when you want to trigger events off the dp being reported without any change in value since the last report, and should always be paired with `optional: true`.  In this case, it seems unnecessary.

Reply:
Ok, removed persist. I can't make them optional since they define how this device is detected, if they are optional then this device type will pop up for every discovered device.